lib/stack-functions

```diff
@@ -448,4 +471,10 @@ _stack_params_arg() {
   fi
 }
 
+_get_stack_capabilities() {
+  # determine what (if any) capabilities a given stack was deployed with
+  local capabilities=$(aws cloudformation describe-stacks --stack-name "$1" --output text | awk /CAPABILITIES/'{print $2}')
```

Comment:
This could be `aws cloudformation describe-stacks --stack-name "shop-staging" --query 'Stacks[].Capabilities' --output text`.

Reply:
https://github.com/realestate-com-au/bash-my-aws/pull/111/commits/69a3b3e6409b4efda07e4979cf34146686bd14dc